Godeps/Godeps.json

```diff
@@ -236,23 +236,28 @@
 		},
 		{
 			"ImportPath": "github.com/coreos/go-systemd/activation",
-			"Comment": "v5",
-			"Rev": "7b2428fec40033549c68f54e26e89e7ca9a9ce31"
+			"Comment": "v6-4-g0f21494",
```

Comment:
@lucab I see you tagged the last version of go-systemd. Could you tag yet another version for this PR?